src/pages/GamePage.js

```diff
@@ -1,42 +1,65 @@
 import 'App.css';
 import 'style.css';
-import LeagueContainer from 'components/LeagueContainer';
+import LeagueCard from 'components/LeagueCard';
+import { Carousel } from '@trendyol-js/react-carousel';
 import { useParams } from 'react-router-dom';
 import { useState, useEffect } from 'react';
 import { http } from 'http';
+import SeriesContainer from 'components/SeriesContainer';
+
+function handleClick() {
+  window.location.href = '/';
+}
 
 const GamePage = () => {
   const params = useParams();
   const [gameName, setGameName] = useState(params.gameName);
   const [leagues, setLeagues] = useState([]);
+  const [resize, setResize] = useState(window.innerWidth);

```

Comment:
We could make a hook out of this. It's a bit hard to figure out why this is needed just from reading the code. So I think maybe moving it into a `hook` would make it a bit easier to understand. Something like useWindowChange